### ISSUE TITLE
babel/object-curly-spacing false positive with comments

### ISSUE THREAD

**Issue by reintroducing** [babel/eslint-plugin-babel#180]
I just upgraded to eslint 6.0.1 and this is triggering a false positive for me:
`daytime_only: { // eslint-disable-line camelcase`

```
  ✘  https://google.com/#q=babel%2Fobject-curly-spacing  There should be no space after '{'
  ../../../../src/js/filters/filters-utils.js:42:19
```

I originally posted this here https://github.com/eslint/eslint/issues/11656 but they pointed me here and I'm getting lost in a trail of eslint plugins and repos :)

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [@babel/eslint-plugin has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
